hoodie-common/src/main/java/com/uber/hoodie/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -45,7 +45,7 @@
  * This class can be serialized and de-serialized and on de-serialization the FileSystem is re-initialized.
  */
 public class HoodieArchivedTimeline extends HoodieDefaultTimeline {
-    private static final String HOODIE_COMMIT_ARCHIVE_LOG_FILE = "commits.archived";
+    private static final String HOODIE_COMMIT_ARCHIVE_LOG_FILE = "commits";
```

Comment:
So for archiving commits, I'm introducing a new extension called ".archive" as you saw in HoodieLogFile. If we have that extension and keep commits.archived, the resulting file name will be like commits.archived_.archive.1. I don't think this name makes sense.